testing/vcs/test_vcs_vectors_robinson.py

```diff
@@ -0,0 +1,24 @@
+import vcs, cdms2, numpy, os, sys
```

Comment:
Ah I see. Thanks @jbeezley. I will push a fix to follow the PEP8 stype in my next branch as I need to take care of doWrap as well.